DOCS.md

```diff
@@ -107,8 +107,9 @@ The following parameters are used to configure the `rebuild` action:
 
 The following parameters are used to configure the `flush` action:
 
-| Name    | Description                          | Required | Default |
-| ------- | ------------------------------------ | -------- | ------- |
+| Name  | Description                              | Required | Default |
+| ----- | ---------------------------------------- | -------- | ------- |
+| `age` | delete the objects past a specific age   | `false`  | `14`    |
```

Comment:
we should add a unit? is it `days`?

Reply:
Yeah, the default is days. I can add a `d` char to it

Reply:
@wass3r @kneal 

We could always make it a configurable option? That is, make not only the amount of days configurable but the actual unit of the `age` field.

To elaborate:

Currently, the `age` field is a string

https://github.com/go-vela/vela-s3-cache/blob/a1acd4852d6dd332fef0d67aa79d85b7f1e0f622/cmd/vela-s3-cache/main.go#L58-L62

If we updated that to a `Duration` flag, then that would allow customers to pass the quantity and the unit of measure. Some examples below:

* `14d` - sets the field to 14 days
* `1h` - sets the field to 1 hour
* `2m` - sets the field to 2 minutes

I know this is beyond the scope of MVP but what are your thoughts on that?

Reply:
I like that. We can toss that change in a new PR. Will this work as is if you supply `14d`, ie. there are no other code change necessary?

Reply:
Okay, so what I'm getting is better docs and new features. So what did y'all want to do with this one?

Reply:
For the purpose of this PR, just clarifying the unit. If `14d` works without any other changes, I think we're good.